[
  {
    "description": "",
    "elements": [
      {
        "after": [
          {
            "match": {
              "location": "stepdefs.rb:5"
            },
            "result": {
              "duration": 119000,
              "error_message": "Failing after hook (RuntimeError)\n./stepdefs.rb:6:in `After'",
              "status": "failed"
            }
          }
        ],
        "description": "",
        "id": "failing-after-hook;s1",
        "keyword": "Scenario",
        "line": 3,
        "name": "S1",
        "steps": [
          {
            "keyword": "Given ",
            "line": 4,
            "match": {
              "location": "stepdefs.rb:13"
            },
            "name": "this step passes",
            "result": {
              "duration": 32000,
              "status": "passed"
            }
          }
        ],
        "tags": [
          {
            "line": 2,
            "name": "@failing_after"
          }
        ],
        "type": "scenario"
      },
      {
        "after": [
          {
            "match": {
              "location": "stepdefs.rb:5"
            },
            "result": {
              "duration": 106000,
              "error_message": "Failing after hook (RuntimeError)\n./stepdefs.rb:6:in `After'",
              "status": "failed"
            }
          }
        ],
        "description": "",
        "id": "failing-after-hook;s1",
        "keyword": "Scenario",
        "line": 3,
        "name": "S1",
        "steps": [
          {
            "keyword": "Given ",
            "line": 4,
            "match": {
              "location": "stepdefs.rb:13"
            },
            "name": "this step passes",
            "result": {
              "duration": 12000,
              "status": "passed"
            }
          }
        ],
        "tags": [
          {
            "line": 2,
            "name": "@failing_after"
          }
        ],
        "type": "scenario"
      },
      {
        "after": [
          {
            "match": {
              "location": "stepdefs.rb:5"
            },
            "result": {
              "duration": 291000,
              "error_message": "Failing after hook (RuntimeError)\n./stepdefs.rb:6:in `After'",
              "status": "failed"
            }
          }
        ],
        "description": "",
        "id": "failing-after-hook;s1",
        "keyword": "Scenario",
        "line": 3,
        "name": "S1",
        "steps": [
          {
            "keyword": "Given ",
            "line": 4,
            "match": {
              "location": "stepdefs.rb:13"
            },
            "name": "this step passes",
            "result": {
              "duration": 17000,
              "status": "passed"
            }
          }
        ],
        "tags": [
          {
            "line": 2,
            "name": "@failing_after"
          }
        ],
        "type": "scenario"
      }
    ],
    "id": "failing-after-hook",
    "keyword": "Feature",
    "line": 1,
    "name": "failing-after-hook",
    "uri": "../../features/failing-after-hook.feature"
  }
]
